lib/matplotlib/axes/_base.py

```diff
@@ -1010,14 +1004,13 @@ def cla(self):
             y0, y1 = self._sharey.get_ylim()
             self.set_ylim(y0, y1, emit=False, auto=None)
             self.yaxis._scale = mscale.scale_factory(
-                    self._sharey.yaxis.get_scale(), self.yaxis)
+                     self._sharey.yaxis.get_scale(), self.yaxis)
```

Comment:
Here too.

Reply:
Those are accidental, and I will revert them.